integration/copy_test.go

```diff
@@ -15,17 +15,24 @@ import (
 )
 
 func init() {
-	check.Suite(&CopySuite{})
+	check.Suite(&CopySuite{
+		ss: &SkopeoSuite{},
+	})
 }
 
 const v2DockerRegistryURL = "localhost:5555" // Update also policy.json
 
 type CopySuite struct {
+	ss       *SkopeoSuite
 	cluster  *openshiftCluster
 	registry *testRegistryV2
 	gpgHome  string
 }
 
+func (s *CopySuite) SetUpTest(c *check.C) {
+	s.ss.SetUpTest(c)
```

Comment:
`SkopeoSuite` is setting up a fresh set of _five_ registries for _every single test_.  That seems a fair bit of unnecessary overhead, especially when `CopySuite.SetUpSuite` is already setting up a v2 registry.